packages/feeds-client/src/feed/feed.ts

```diff
@@ -51,6 +51,7 @@ import type {
   PagerResponseWithLoadingStates,
 } from '../types';
 import { checkHasAnotherPage, Constants, uniqueArrayMerge } from '../utils';
+import { handleActivityMarked } from './event-handlers/activity/handle-activity-marked';
```

Comment:
Export this from the `event-handlers/index.ts`, there's an import on [line 44](https://github.com/GetStream/stream-feeds-js/blob/b61af6b52f8783b1c8766233d82907684191679d/packages/feeds-client/src/feed/feed.ts#L44).